code/leslie/Python/lab07.py

```diff
@@ -0,0 +1,100 @@
+import math # because I'm using math.ceil() to round up the answer
+
+with open('gettysburg_address.txt', 'r') as f:
+    contents = f.read()
+    words = contents.split() # Splits into words
+    chars = contents.replace(' ', '') #reads content and replaces spaces with NO spaces to get accurate character count
+    sentences = contents.split('.') # splits at periods
+#print(words)
+word_count = len(words)
+print("Number of words: ", word_count)
+
+char_count = len(chars)
+print("Number of characters: ", char_count)
+
+sentence_count = len(sentences)
+print("Number of sentences: ", sentence_count)
+
+
+first = (char_count/word_count) * 4.71
+#print(first)
+second = (word_count/ sentence_count) * 0.5
+#print(second)
+
+formula = first + second - 21.43
+print("Answer pre-ceiling: ", formula)
+ari_num = math.ceil(formula)
+print("Should be rounded up: ", ari_num)
+
+ari_scale = {
```

Comment:
Great use of formatting of code so it's easier to read.

Separating dictionary elements over multiple lines.